packages/parcel-optimizer/src/index.ts

```diff
@@ -71,7 +71,7 @@ export default new Optimizer<ParcelOptimizerOpts, unknown>({
           .use(
             insertAt({
               selector: 'head',
-              append: '<style>' + stylesheet + '</style>',
+              append: '<style title="compiled">' + stylesheet + '</style>',
```

Comment:
Btw, do we inline CSS also in webpack? I don't think I have seen `inlineCss` for it